vtkext/private/module/vtkF3DImguiActor.cxx

```diff
@@ -471,3 +471,10 @@ void vtkF3DImguiActor::EndFrame(vtkOpenGLRenderWindow* renWin)
   ImGui::Render();
   this->Pimpl->RenderDrawData(renWin, ImGui::GetDrawData());
 }
+
+//----------------------------------------------------------------------------
+void vtkF3DImguiActor::SetDeltaTime(double time)
+{
+  ImGuiIO& io = ImGui::GetIO();
```

Comment:
You can add `this->Superclass::SetDeltaTime(time);` to cover the parent class method.

Reply:
Lets not add code for coverage sake.

Reply:
At least do it for consistency, it's how it's done in all other methods.

Reply:
Ill remove it from other method (in another PR) then. I really dont like adding code that has no effect. Its very confusing.

Reply:
https://github.com/f3d-app/f3d/pull/1791

Reply:
I think it doesn't hurt but ok